### PR TITLE
Fix spacing and typos

### PR DESCRIPTION
**Pull request by mitogh**
Continue the update and fix of multi lines on the docs.

### CODE REVIEW

**Comment by lucatume** [comment]
This is more of a question about the "style" you're using in this PR and the previous one: why so many blank lines? Blank lines are meant to break up not-so-related paragraphs but adding one for each line break seems an excess to me.

Could you provide some context?

File: docs/installation.md [hunk 1]
```diff
@@ -1,31 +1,49 @@
 ## Installation
 
 ### Where should I install wp-browser?
-As a rule-of-thumb wp-browser should be installed in the root folder of your project.  
+
+As a rule-of-thumb wp-browser should be installed in the root folder of your
```
Comment (lucatume) [inline docs/installation.md]: Hard-wrap here and not elsewhere? On purpose? Why?

File: docs/faq.md [hunk 2]
```diff
@@ -1,73 +1,119 @@
...
+(levels-of-testing.md#acceptance-tests) on the site frontend and a `backend` suite running acceptance tests on the site backend.  
+
 Think of suites as a tool to organize your tests: there's a good measure between too organized and not organized at all.
 
 ### I've used PHPUnit before for my unit tests, can I reuse that knowledge and code with wp-browser?
+
 Yes.
-Codeception uses PHPUnit as one of its main components and can run PHPUnit tests with little or no modification.  
+
+Codeception uses PHPUnit as one of its main components and can run PHPUnit tests
```
Comment (lucatume) [inline docs/faq.md]: Hard-wrap here and not elsewhere? On purpose? Why?

File: docs/faq.md [hunk 3]
```diff
@@ -1,73 +1,119 @@
...
 An example is one where a test for the handling of Ajax requests sets the `DOING_AJAX` constant: this will be now set for **any** test after the one that set it thus breaking, or worse altering, all the following ones.
+
 So, in short, **run each suite separately**.
 
 ### Can I have more than one suite of one kind?
+
 Yes, you should.  
-As an example you might have a `frontend` suite running [acceptance tests](levels-of-testing.md#acceptance-tests) on the site frontend and a `backend` suite running acceptance tests on the site backend.  
+
+As an example you might have a `frontend` suite running [acceptance tests]
```
Comment (lucatume) [inline docs/faq.md]: Hard-wrap here and not elsewhere? On purpose? Why?

File: docs/installation.md (hunk 1 above)
Comment (mitogh) [reply docs/installation.md]: This one is mainly for clearly identify the next section but does not have any impact on the output.

File: docs/faq.md (hunk 2 above)
Comment (mitogh) [reply docs/faq.md]: Intentional, explained on final comment.

File: docs/faq.md (hunk 3 above)
Comment (mitogh) [reply docs/faq.md]: Intentional, explained on final comment.

**Comment by mitogh** [comment]
Thanks for the questions. Basically where this instances take place this is the current behavior.

Screen Shot 2019-07-06 at 16 55 10

And this one is the new one.

Screen Shot 2019-07-06 at 16 57 34

As you can see second one provides a bit more cleaner way to separate each paragraph, which from my point of view is easier to read and spot specific content areas in case of looking for some specific data. 


Let me know if you think otherwise.

**Comment by lucatume** [comment]
> Thanks for the questions. Basically where this instances take place this is the current behavior.
> [...]

Ok, I see your point and I agree, thanks for taking the time to explain.
I will fix the doc issue that is not passing the documentation test, push and merge.

Again: thanks.

**Pull request merged**